README.md

```diff
@@ -217,13 +217,13 @@ everyauth service set slack \
   --clientSecret "{your-client-secret}
 ```
 
-Configuration parameters of a service are durably stored by EveryAuth as part of your Fusebit account.
+EveryAuth durably stores configuration parameters of a service as part of your Fusebit account.
```

Comment:
... stores *the* configuration parameters...

Reply:
Fixed